README.md

```diff
@@ -2,19 +2,105 @@
 
 A .NET Rollbar Client that is not ASP.NET specific.
 
-## Install
+## Installation
 
 Nuget Package Manager:
 
     Install-Package Rollbar
 
-## Basic Usage
+## Upgrading to v1.0.0 from earlier versions
 
```

Comment:
I'm not certain what information to give our users here as upgrading instructions.

Reply:
Since, the v1 supports singleton-like instance that is similar to the static implementation of Rollbar in v0, hence, the quickest update path would be to:

1. change older Rollbar configuration:  Rollbar.Init(new RollbarConfig("POST_SERVER_ACCESS_TOKEN"))
   using something like:
  const string postServerItemAccessToken = "POST_SERVER_ACCESS_TOKEN";
 
 RollbarLocator.RollbarInstance
     .Configure(new RollbarConfig(postServerItemAccessToken) { Environment = "proxyTest" }) ; 

2. Every place that used to send reports via Rollbar.Report(Exception) or Rollbar.Report(string) has to be replaces with something like: RollbarLocator.RollbarInstance.Error(new Exception("trying out the TraceChain", new NullReferenceException())) or RollbarLocator.RollbarInstance.Info("Basic info log example.")